src/main/java/net/logstash/logback/pattern/AbstractJsonPatternParser.java

```diff
@@ -160,6 +199,19 @@ protected Double transform(final String value)  throws NumberFormatException {
         }
     }
 
+    protected static class AsJsonValueTransformer<Event> extends AbstractAsObjectTransformer<JsonNode, Event> {
+        private static final ObjectMapper mapper = new ObjectMapper()
+                .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
+                .enable(JsonGenerator.Feature.ESCAPE_NON_ASCII);
+
+        public AsJsonValueTransformer(final ValueGetter<String, Event> generator) {
+            super(generator);
+        }
+
+        protected JsonNode transform(final String value) throws IOException {
+            return mapper.readTree(value);
```

Comment:
Hmm.  I'm wondering if this could be refactored slightly differently to avoid the json parsing.  I mean, as I understand it,  you ultimately just want to write the value to the generator as-is, right?.  e.g. `generator.writeRaw(String)`.  

The problem with the current design, is that `ComputableValueWriter` always calls `writeObject`.  So, if you actually wanted to call `writeRaw` instead of `writeObject`, you'd probably need to remove the `ValueGetter` concept, and just do the value retrieval, transformation, and writing in a `NodeWriter`.

In addition, I'm a little worried about the static configuration of the `ObjectMapper` used to parse the value.   i.e. what happens if this default configuration doesn't work for somebody's JSON data.  If parsing could be avoided, then there would be no need for the `ObjectMapper`.

Your thoughts?